src/Artsy/Router/buildServerApp.tsx

```diff
@@ -44,15 +53,11 @@ export function buildServerApp(config: RouterConfig): Promise<Resolve> {
           })
         )
 
-        if (redirect) {
-          resolve({
-            redirect,
-            // TODO: The docs seem to indicate that if there’s a redirect there
-            //       will not be a status.
-            //       https://github.com/4Catalyzer/found#server-side-rendering
-            status,
-          })
+        if (isRedirect(farceResult)) {
+          resolve(farceResult)
           return
+        } else {
+          farceResult.element
```

Comment:
`return farceResult.element`?

Reply:
Actually I think this was me just testing the typing, namely that using the isRedirect type guard would narrow the union down correctly. So this can be removed.